constellations/orion/src/repos/achieve_repo_test.go

```diff
@@ -115,6 +115,17 @@ func TestSelectAllGroupedByYear(t *testing.T) {
 			domains.NullTime{},
 			2021,
 			"1600 on SAT",
+			2,
+		).
+		AddRow(
+			3,
+			now,
+			now,
+			domains.NullTime{},
+			domains.NullTime{},
+			2021,
+			"800 on SAT Math",
+			1,
```

Comment:
as described from a DM,

when you're in `repo_test` there isn't any database logic. The `rows` variable here is supposed to represent whatever you want the database to return. So in the repo_test, there's no re-ordering of any sorts done (no database to do that). Instead, we have to specify the database is going to return *this*.